Azure/azure-cli
# {AKS} Update region mapping list for Container Insights to include latest regions

**Description**

This PR is updating the AKS region to Log Analytics region mapping for the Container Insights service.
Both AKS and Log analytics have added new regions since the last update and this PR makes sure that we don't default to eastus for those new regions and actually map it to its proper location.

**Testing Guide**

Scenario for testing:
Create an AKS cluster in one of the new regions with monitoring enabled (e.g. make sure the resource group is in a new region like: Norway East)
command -> 
az aks create --resource-group myResourceGroup --name myAKSCluster --node-count 1 --enable-addons monitoring

Once this command finishes you should see that monitoring is enabled for this cluster and its sending data to the default norwayeast log analytics workspace.

You can check this by doing an 'az aks show --resource-group myResourceGroup --name myAKSCluster' and checking under the omsagent addon for the log analytics workspace id.

**History Notes**


[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
